src/styles.js

```diff
@@ -1,19 +1,39 @@
 import {createGlobalStyle} from 'styled-components';
 
 export const GlobalStyle = createGlobalStyle`
+/* abril-fatface-regular - latin */
+@font-face {
+  font-family: 'Abril Fatface';
+  font-style: normal;
+  font-weight: 400;
+  src: url('../fonts/abril-fatface-v19-latin-regular.eot'); /* IE9 Compat Modes */
+  src: local(''),
+       url('../fonts/abril-fatface-v19-latin-regular.eot?#iefix') format('embedded-opentype'), /* IE6-IE8 */
+       url('../fonts/abril-fatface-v19-latin-regular.woff2') format('woff2'), /* Super Modern Browsers */
+       url('../fonts/abril-fatface-v19-latin-regular.woff') format('woff'), /* Modern Browsers */
+       url('../fonts/abril-fatface-v19-latin-regular.ttf') format('truetype'), /* Safari, Android, iOS */
+       url('../fonts/abril-fatface-v19-latin-regular.svg#AbrilFatface') format('svg'); /* Legacy iOS */
+}
 	*,
 	*::before,
 	*::after {
 		box-sizing: border-box;
 	}
 
 	html {
-		font-family: Arial, Helvetica, sans-serif;
 		font-size: 16px;
 	}
 
 	body {
 		margin: 0;
+		background-image: url('./../background-test2.jpg');
```

Comment:
If you decide to use this background, we recommend that you rename the file (without test-2)
```suggestion
		background-image: url('./../background.jpg');
```